apps/src/turtle/turtle.js

```diff
@@ -1510,7 +1524,7 @@ Artist.prototype.step = function (command, values, options) {
       result = this.calculateSmoothAnimate(options, distance);
       tupleDone = result.tupleDone;
       this.visualization.setHeading(heading);
-      this.visualization.jumpForward(result.distance);
+      this.jumpForward(result.distance);
```

Comment:
This changed with Josh's recent refactoring, it should be `this.visualization.jumpForward(...`